haydenfetch.mk

```diff
@@ -0,0 +1,42 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS      += haydenfetch
+HAYDENFETCH_VERSION := 1.0.0
+DEB_HAYDENFETCH_V   ?= $(HAYDENFETCH_VERSION)
+
+haydenfetch-setup: setup
+	if [ ! -d "$(BUILD_WORK)/haydenfetch" ]; then \
+	git clone https://github.com/asdfugil/haydenfetch.git  $(BUILD_WORK)/haydenfetch; \
+	cd $(BUILD_WORK)/haydenfetch; \
+	git fetch origin; \
+	git reset --hard origin/master; \
+	git checkout "3d16901a8d0e8699b7d1484f135d7c2aad402ade"; \
+	fi
```

Comment:
I feel like, because you own this repository, you should make a tag and use that implementation of downloading a tar rather than using this, which depends on ``git``, which is not a Procursus dependency.